chartjs-plugin-zoom.js

```diff
@@ -390,15 +388,16 @@ var zoomPlugin = {
 					var chartDistance = chartArea.right - chartArea.left;
 					var zoom = 1 + ((chartDistance - dragDistance) / chartDistance );
 
+					// Remove drag start and end before diagram update to stop drawing selected area
```

Comment:
can you change "diagram" to "chart" in this comment to match the terminology we normally use